store/storetest/team_store.go

```diff
@@ -869,7 +871,118 @@ func testTeamCount(t *testing.T, ss store.Store) {
 	require.Equal(t, countNotIncludingDeleted+1, countIncludingDeleted)
 }
 
-func testGetMembersOrder(t *testing.T, ss store.Store) {
+func testGetMembersExcludedDeletedUsers(t *testing.T, ss store.Store) {
+	teamId1 := model.NewId()
+	teamId2 := model.NewId()
+
+	u1 := &model.User{Email: MakeEmail()}
+	u2 := &model.User{Email: MakeEmail(), DeleteAt: int64(1)}
+	u3 := &model.User{Email: MakeEmail()}
+	u4 := &model.User{Email: MakeEmail(), DeleteAt: int64(3)}
+	u5 := &model.User{Email: MakeEmail()}
+	u6 := &model.User{Email: MakeEmail(), DeleteAt: int64(5)}
+
+	u1, err := ss.User().Save(u1)
+	require.Nil(t, err)
+	u2, err = ss.User().Save(u2)
+	require.Nil(t, err)
+	u3, err = ss.User().Save(u3)
+	require.Nil(t, err)
+	u4, err = ss.User().Save(u4)
+	require.Nil(t, err)
+	u5, err = ss.User().Save(u5)
+	require.Nil(t, err)
+	u6, err = ss.User().Save(u6)
+	require.Nil(t, err)
+
+	m1 := &model.TeamMember{TeamId: teamId1, UserId: u1.Id}
+	m2 := &model.TeamMember{TeamId: teamId1, UserId: u2.Id}
+	m3 := &model.TeamMember{TeamId: teamId1, UserId: u3.Id}
+	m4 := &model.TeamMember{TeamId: teamId1, UserId: u4.Id}
+	m5 := &model.TeamMember{TeamId: teamId1, UserId: u5.Id}
+	m6 := &model.TeamMember{TeamId: teamId2, UserId: u6.Id}
+
+	_, err = ss.Team().SaveMember(m1, -1)
+	require.Nil(t, err)
+	_, err = ss.Team().SaveMember(m2, -1)
+	require.Nil(t, err)
+	_, err = ss.Team().SaveMember(m3, -1)
+	require.Nil(t, err)
+	_, err = ss.Team().SaveMember(m4, -1)
+	require.Nil(t, err)
+	_, err = ss.Team().SaveMember(m5, -1)
+	require.Nil(t, err)
+	_, err = ss.Team().SaveMember(m6, -1)
+	require.Nil(t, err)
+
+	// Gets users ordered by UserName
+	ms, err := ss.Team().GetMembers(teamId1, 0, 100, nil, &model.TeamMembersGetOptions{ExcludeDeletedUsers: true})
+	require.Nil(t, err)
+	assert.Len(t, ms, 3)
```

Comment:
nit: Could we assert the actual team members being returned here? If the logic happened to be reversed, we'd still get three, but accidentally get the deleted three, and the unit test would never catch this.

Reply:
`require.ElementsMatch...` would be great here

Reply:
Thanks for this recommendation. I have used what Farhan recommended.